hugr-py/docs/api-docs/_templates/autosummary/class.rst

```diff
@@ -0,0 +1,37 @@
+..
+   Custom class template to make sphinx-autosummary list the full API doc after
+   the summary. See https://github.com/sphinx-doc/sphinx/issues/7912
+
+{{ name | escape | underline}}
+
+.. currentmodule:: {{ module }}
+
+.. autoclass:: {{ objname }}
+   :members:
+   :show-inheritance:
+   :inherited-members:
```

Comment:
Including all the inherited members does mean that the docs are blown up and it can be difficult to find what one wants. For example, the page for `Else` (`hugr/hugr-py/docs/build/api-docs/generated/hugr.cond_loop.Else.html`) includes all methods and inherited methods in alphabetical order, which is quite a long list! Not sure if this could be better organized.

Reply:
yeah, is it obvious to a user that they should look at the base class methods when they go to check something about else?

I guess the most common use case is search, which isn't too affected by including everything.